src/Symfony/Component/Process/Process.php

```diff
@@ -1264,7 +1303,7 @@ protected function buildCallback(callable $callback = null)
         if ($this->outputDisabled) {
             return function ($type, $data) use ($callback) {
                 if (null !== $callback) {
-                    \call_user_func($callback, $type, $data);
+                    return $callback($type, $data);
```

Comment:
does this finally work for **all** kind of callables in PHP 7.1+ ? In PHP 5.6, I know there was still 1 kind of callable failing this.

Reply:
I was not aware of that fact. No idea.

Reply:
Let's revert this change, no big deal (same below)